### PR TITLE
NetworkControllerDiagnostics PowerShell reference update

### PR DESCRIPTION
**Pull request by TimShererWithAquent**
[1882593](https://dev.azure.com/mseng/TechnicalContent/_workitems/edit/1882593)

1 new cmdlet
1 new param
Minor changes in 4 files

Cmdlets had more content than I realized.

### CODE REVIEW

**Comment by TimShererWithAquent** [comment]
Hi @tiburd - this one is ready for review.

**Review by get-itips — APPROVED** [review]

File: docset/winserver2022-ps/networkcontrollerdiagnostics/Get-NetworkControllerManagedDevices.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+```
+
+## DESCRIPTION
+The **Get-NetworkControllerManagedDevices** cmdlet gets managed devices for a Network Controller. The cmdlet aggregates results for devices from **Get-NetworkControllerServer**, **Get-NetworkControllerVirtualServer**, **Get-NetworkControllerGateway**, and **Get-NetworkControllerLoadBalancerMux**, and then presents the connections information for every device.
+
+## EXAMPLES
+
+### Example 1: Get managed devices
+```powershell
+Get-NetworkControllerManagedDevices -RestURI  "https://nc.contoso.cloud.com"
```
Comment (tseward) [inline docset/winserver2022-ps/networkcontrollerdiagnostics/Get-NetworkControllerManagedDevices.md:30]: Looks like there is an extra space between -RestURI and the URI. Is that intentional?

Comment (TimShererWithAquent) [reply docset/winserver2022-ps/networkcontrollerdiagnostics/Get-NetworkControllerManagedDevices.md:30]: Nope - I'll fix it

File: docset/winserver2022-ps/networkcontrollerdiagnostics/Get-NetworkControllerManagedDevices.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+Required: True
+Position: 0
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (dariomws) [inline docset/winserver2022-ps/networkcontrollerdiagnostics/Get-NetworkControllerManagedDevices.md:86]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```

File: docset/winserver2022-ps/networkcontrollerdiagnostics/Get-NetworkControllerManagedDevices.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+## INPUTS
+
+### None
+
+## OUTPUTS
+
+### System.Object
```
Comment (dariomws) [inline docset/winserver2022-ps/networkcontrollerdiagnostics/Get-NetworkControllerManagedDevices.md:94]:
```suggestion
### System.Object

```

**Review by dariomws — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Comment by TimShererWithAquent** [comment]
@tiburd This one looks like it's ready to merge.

**Pull request merged**